src/app/app.module.ts

```diff
@@ -5,6 +5,7 @@ import { AppComponent } from './app.component';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { PagesModule } from './pages/pages.module';
 import { ComponentsModule } from './components/components.module';
+import {  } from '@fortawesome/free-solid-svg-icons';
```

Comment:
should this be import `{ FontAwesomeModule } from '@fortawesome/angular-fontawesome';`

Reply:
I used this package a few times, but I don't think you need to actually import it into a component to use it.

An import at the top of the styles.scss file is enough to get the icons accessible using the normal css classes:

```
@import "@fortawesome/fontawesome-free/css/all.min.css";
```

Reply:
This line can be removed now.

Reply:
Ah, it's marked as outdated :-)